packages/ui/src/primitives/tailwind/Modal/index.tsx

```diff
@@ -43,26 +43,41 @@ export interface ModalProps {
   submitButtonDisabled?: boolean
   closeButtonText?: string
   submitButtonText?: string
+  startSubmitIcon?: ReactNode
+  actionsOnHeader?: boolean
   onClose?: (isHeader: boolean) => void
   onSubmit?: () => void
 }
 
 export const ModalHeader = ({
   title,
+  actionsOnHeader,
+  startSubmitIcon,
+  submitButtonText,
+  onSubmit,
   onClose
 }: {
   closeIcon?: boolean
   title?: string
+  startSubmitIcon?: ReactNode
+  actionsOnHeader?: boolean
+  submitButtonText?: string
+  onSubmit?: () => void
   onClose?: (isHeader: boolean) => void
 }) => {
   // sticky top-0 z-10 bg-theme-surface-main
+  const { t } = useTranslation()
   return (
-    <div className="relative flex items-center justify-center border-b border-b-theme-primary px-6 py-5">
+    <div className="relative flex items-center justify-between border-b border-b-theme-primary px-6 py-5">
```

Comment:
according to the admin figma designs, text should be centered in the modal

https://www.figma.com/design/9F1J9ZuxkCbimWUzjzDzTm/Ready-for-dev%3A-Dashboard-%26-Admin?node-id=1-11949&t=UF1WY267SMoHc2V6-0

currently we are having it like this
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/b72298ce-f052-4762-9c9b-d2c1527ad0ab)

Reply:
You can ask this from UI team on figma as to what should be the unified standard.

https://www.figma.com/design/Ds2NEvvz9nqddltP3lsKoP/Design-Changes?node-id=0-1&t=YLp6WBatFXncmfY7-0